go.mod

```diff
@@ -14,6 +14,8 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.13.0
 	github.com/vishvananda/netlink v1.1.0
+	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a
```

Comment:
pls keep all pkgs update in its own commit

Reply:
This would break bisect. I grouped the syslog implementation and vendor of the packages as one commit deliberately.